packages/actor-rdf-resolve-quad-pattern-federated/lib/FederatedQuadSource.ts

```diff
@@ -221,7 +226,11 @@ export class FederatedQuadSource implements IQuadSource {
       // Prepare the context for this specific source
       let context: IActionContext = getDataSourceContext(source, this.contextDefault);
       context = context.set(KeysRdfResolveQuadPattern.source,
-        { type: getDataSourceType(source), value: getDataSourceValue(source) });
+        {
```

Comment:
The changes here won't be needed anymore when using the context to store this information.